feat(kmp): update getting started and add manual setup page (#11497)
getsentry/sentry-docs: docs/platforms/kotlin-multiplatform/initialization-strategies.mdx

---

> title: Initialization Strategies

Reviewer: ```suggestion
title: Initialization Options
```

> description: "Strategies for initializing the Kotlin Multiplatform SDK."

Reviewer: ```suggestion
description: "Different options for initializing the Kotlin Multiplatform SDK."
```

sidebar_order: 2
---

> When it comes to initializing a Kotlin Multiplatform SDK, there are multiple strategies to consider:

Reviewer: ```suggestion
When it comes to initializing a Kotlin Multiplatform SDK, there are multiple options to consider:
```

> Regardless of the initialization strategy you choose, it's important to place the initialization code at an early lifecycle stage of your application to ensure the SDK is set up correctly before it is used.

Reviewer: ```suggestion
Regardless of the initialization option you choose, it's important to place the initialization code at an early lifecycle stage of your application to ensure the SDK is set up correctly before it is used.
```
